.env.production.example

```
@@ -2,6 +2,9 @@ API_DOMAIN=api.zentra-app.pro
 LETSENCRYPT_EMAIL=admin@zentra-app.pro
 DATABASE_URL=
 DO_HOST=
+ARGENTINA_DATOS_API_URL=https://api.argentinadatos.com
+ARS_USD_EXCHANGE_HOUSE=blue
+CRON_SYNC_ARS_USD_RATE=0 0,12 * * *
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The CRON_SYNC_ARS_USD_RATE key should go before the DATABASE_URL key

(UnorderedKey)